schema.graphql

```diff
@@ -107,7 +107,7 @@ type Query {
   location(
     # City name of location
     cityName: String!
-  ): [Location!]!
+  ): Location!
```

Comment:
looks like this got squashed into the wrong commit. This should be on e80c119f6b474c8f67d943de41bfaa0b91138e10

Reply:
Just pushed the correct commits